src/ShallowWrapper.js

```diff
@@ -142,6 +142,16 @@ class ShallowWrapper {
     this.complexSelector = new ComplexSelector(buildPredicate, findWhereUnwrapped, childrenOfNode);
   }
 
+  batchedUpdates(fn) {
```

Comment:
instead of exposing another prototype method, what do you think about using a standalone function?